website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -106,3 +109,53 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt --help`: Lists the commands available for dbt<br />
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
+
+
+## FAQs
+<expandable alt_header="How to create a .dbt directory and move your file">
+
+If you've never had a `.dbt` directory, you should perform the following recommended steps to create one. If you already have `.dbt` directory, then you should be able to move the `dbt_cloud.yml` file into it.
+
+<Tabs>
+<TabItem value="Create a .dbt directory">
+
+  1. Clone your dbt project repository locally.
+  2. Use the `cd` command to change directories and navigate to the root of your filesystem. (For example, if you're a Mac user it might be `Macintosh HD -> Users -> YOUR_USERNAME`):
+
+     ```bash
+     cd ~/Users
+      ```
+  3. Use the `mkdir` command followed by the name of the folder you want to create. To create a `.dbt` folder, use:
+
+     ```bash
+     mkdir .dbt
+     ```
+
+This will create a `.dbt` folder in the current directory. 
+
+For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. However, to view hidden files and folders press Command + Shift + G. 
```

Comment:
```suggestion
For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. To view hidden files and folders, press Command + Shift + G. 
```